_schemaV2.graphql

```diff
@@ -2542,6 +2542,9 @@ type ArtworkVersion implements Node {
   # The artists related to this Artwork Version
   artists: [Artist]
 
+  # The Artwork Version attribution class
+  attribution_class: String
```

Comment:
Can we use camelCase here?

Reply:
fixed: https://github.com/artsy/metaphysics/pull/4671